BUILD

```diff
@@ -29,24 +29,12 @@ java_image(
     ],
 )
 
-pkg_tar(
-    name = "proxy_config",
-    package_dir = "/config",
-    srcs = [
-        "config.yml"
```

Comment:
should default config be removed from repo? or maybe included in docker image with ability to use it via some options?

Reply:
Maybe it is sufficient to remove the command line option? Not sure how it works to supply extra arguments to the bazel java_image?

Reply:
from jenkins side there's containerTemplate.args https://github.com/jenkinsci/kubernetes-plugin

Reply:
I also have to change proxy.sh. I intend to send everything through the proxy, and then copy the cache to ~/.m2/repository . This would then make maven_install.json be independent of where bazel pin runs.